modules/test/playwright/tests/frontend-data-set-views-web/pages/FDSFragmentPage.ts

```diff
@@ -115,22 +119,6 @@ export class FDSFragmentPage {
 			.waitFor({state: 'visible'});
 	}
 
-	async dragAndDropFragment(itemName: string) {
-		const source = await this.page.getByRole('menuitem', {
-			exact: true,
-			name: itemName,
-		});
-
-		await source.focus();
-		await source.press('Enter');
-		await source.press('Enter');
```

Comment:
This was causing the flaky behavior for me, I am not sure why. Manual double enter worked fine. I rewrote it to do drag & drop.

Reply:
Awesome. Hope that this work in a smooth way.
I remember start tinkering with the Drag & Drop but finding issues due to the side panel (widgets & fragments) hiding part of the drop area. 🤞